ext/reflection/php_reflection.c

```diff
@@ -1595,6 +1595,9 @@ ZEND_METHOD(Reflection, getModifierNames)
 	if (modifiers & ZEND_ACC_FINAL) {
 		add_next_index_stringl(return_value, "final", sizeof("final")-1);
 	}
+	if (modifiers & ZEND_ACC_VIRTUAL) {
+		add_next_index_stringl(return_value, "virtual", sizeof("virtual")-1);
+	}
```

Comment:
Just a note: This is not a great function, as it provides no way of disambiguating flags. We just got lucky again that `ZEND_ACC_VIRTUAL` doesn't clash with any other flags returned from any of the `getModifiers()` methods. We should probably deprecated it and replace it with class-specific counterparts, e.g. `ReflectionProperty::getModifierNames()`.